src/content/docs/snowflake/features/javascript-stored-procedures.md

```diff
@@ -0,0 +1,34 @@
+---
+title: JavaScript Stored Procedures
+description: Get started with JavaScript Stored Procedures in LocalStack for Snowflake
+tags: ["Base"]
+---
+
+## Introduction
+
+JavaScript Stored Procedures uses [Snowflake JavaScript Procedures API](https://docs.snowflake.com/en/developer-guide/stored-procedure/stored-procedures-api). The API consists of JavaScript objects and the methods in those objects. You can declare JavaScript stored procedures, execute SQL via embedded JavaScript, call these using Snowflake’s supported methods.
+
+The methods that we support thus far are:
```

Comment:
Need confirmation on whether this release also supports the following 2 methods:
- resultSet.next
- resultSet.getColumnValue